vertx-sql-client/src/main/java/io/vertx/sqlclient/PoolOptions.java

```diff
@@ -101,21 +129,20 @@ public JsonObject toJson() {
 
   @Override
   public boolean equals(Object o) {
-    if (this == o) return true;
-    if (!(o instanceof PoolOptions)) return false;
-    if (!super.equals(o)) return false;
```

Comment:
Was this intentionally removed?

Reply:
No, the first two line have been reformatted, the third line has been changed and reformatted.